examples/cards.py

```diff
@@ -1,29 +1,89 @@
 from twitter_ads.client import Client
 from twitter_ads.creative import Card
-from twitter_ads.campaign import Tweet
-from twitter_ads.restapi import UserIdLookup
+from twitter_ads.http import Request
 
 
 CONSUMER_KEY = 'your consumer key'
 CONSUMER_SECRET = 'your consumer secret'
-ACCESS_TOKEN = 'access token'
-ACCESS_TOKEN_SECRET = 'access token secret'
-ACCOUNT_ID = 'account id'
+ACCESS_TOKEN = 'user access token'
+ACCESS_TOKEN_SECRET = 'user access token secret'
+ACCOUNT_ID = 'ads account id'
 
 # initialize the client
 client = Client(CONSUMER_KEY, CONSUMER_SECRET, ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
 
 # load the advertiser account instance
 account = client.accounts(ACCOUNT_ID)
 
-# create the card
-name = 'video website card'
-components = [{"type":"MEDIA","media_key":"13_1191948012077092867"},{"type":"DETAILS","title":"Twitter","destination":{"type":"WEBSITE", "url":"http://twitter.com/"}}]
-video_website_card = Card.create(account, name=name, components=components)
+# fetch all
```

Comment:
We should add a separate example for Card Creation

Reply:
Yep its been added below